assets/src/dashboard/animations/utils/keyframes.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const getBounceKeyframes = () => {
+  return `@keyframes bounce {
+    0% { transform: scale(0); }
```

Comment:
Only one nuance here, which we can easily come back to this later. But if we were to specify the [keyframes as JSON](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Keyframe_Formats), we could use the same definitions with Web Animations API, which could be easier to use in the editor mode, e.g. for replay, timeline control, easier pause/resume, etc.